docs/inference_api.md

```diff
@@ -280,18 +280,20 @@ For each loaded model, user can make REST call to URI: /predictions/{model_name}
 ```bash
 curl -O https://s3.amazonaws.com/model-server/inputs/kitten.jpg
 
-curl -X POST http://localhost:8080/predictions/resnet-18 -F "data=@kitten.jpg"
+curl -X POST http://localhost:8080/predictions/resnet-18 -T kitten.jpg
 
 or:
 
-curl -X POST http://localhost:8080/predictions/resnet-18 -T kitten.jpg
+curl -X POST http://localhost:8080/predictions/resnet-18 -F "data=@kitten.jpg"
```

Comment:
Why two examples, wouldn't it be enough to have one example here?

Reply:
Trying show case we accept multiple type of input.